smartvisu/locale.yaml

```diff
@@ -21,4 +21,10 @@ plugin_translations:
     'nicht aktiv':                 {'de': '=', 'en': 'disabled'}
     'erlaubt':                     {'de': '=', 'en': 'enabled'}
     'nicht erlaubt':               {'de': '=', 'en': 'disabled'}
-
+    'Letzte Änderung':             {'de': '=', 'en': 'Last Change'}
+    'Letzte Aktualisierung':       {'de': '=', 'en': 'Last Update'}
+    'Logik':                       {'de': '=', 'en': 'Logic'}
+    'Status':                      {'de': '=', 'en': '='}
+    'Item':                        {'de': '=', 'en': '='}
```

Comment:
Zumindest die Übersetzung für **Item** ist doppelt gemoppelt. Die gibt es doch bereits in den globalen Übersetzungen in bin/locale.yaml. Dort sogar mit französischer Übersetzung, was durch den Eintrag hier jedoch ausgeblendet wird.

Reply:
Guter Hinweis, an das File hab ich nie gedacht.. da könnte man dann wohl noch einige andere Plugins säubern. Hab mal ein paar unnötige Übersetzungen rausgenommen.